br/br-pitr-manual.md

```diff
@@ -83,13 +83,72 @@ The example output only shows the common parameters. These parameters are descri
 Usage example:
 
 ```shell
-tiup br log start --task-name=pitr --pd="${PD_IP}:2379" \
---storage='s3://backup-101/logbackup?access-key=${access-key}&secret-access-key=${secret-access-key}"'
+tiup br log start \
+  --task-name=pitr \
+  --pd="${PD_IP}:2379" \
+  --storage='s3://backup-101/logbackup?access-key=${access-key}&secret-access-key=${secret-access-key}'
 ```
```

Comment:
Please also make the above change in the Chinese PR https://github.com/pingcap/docs-cn/pull/18650/files

Reply:
ah got it!